library/Rules/HexRgbColor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Respect\Validation\Rules;
+
+use Respect\Validation\Validator as v;
+
+class HexRgbColor extends AbstractRule
+{
+    public function validate($input)
+    {
+        if(v::oneOf(v::object(), v::arr(), v::nullValue())->validate($input)){
```

Comment:
As [I already](https://github.com/Respect/Validation/pull/198/files#r22520098) said about this schema, it's not a number, they are 3 concatenated numbers. If you allow integers we can have problems, think about this blue: 000099; when converting to string it will be only `99` not and a string with `000099`.